ws-calculator/src/main/java/org/egov/wscalculation/repository/builder/WSCalculatorQueryBuilder.java

```diff
@@ -155,6 +155,8 @@ public String getConnectionNumberList(String tenantId, String connectionType, Li
 		addClauseIfRequired(preparedStatement, query);
 		query.append(" conn.tenantid = ? ");
 		preparedStatement.add(tenantId);
+		addClauseIfRequired(preparedStatement, query);
```

Comment:
Why is this condition required?

Reply:
We need the list of connections without any nullable connection no, So for that we put this query.